### ISSUE TITLE
websocket error when reverse proxy through nginx

### ISSUE THREAD

**Issue by suckerfish** [m1k1o/neko#116]
```
2021-12-07 23:01:39,730 DEBG 'neko' stdout output:
7:01AM ERR failed to upgrade connection error="websocket: the client is not using the websocket protocol: 'websocket' token not found in 'Upgrade' header" module=websocket
7:01AM WRN failed to upgrade websocket conection error="websocket: the client is not using the websocket protocol: 'websocket' token not found in 'Upgrade' header" module=http
```

Internal access works great. Externally through nginx reverse proxy, I get the above message in the logs when I try to log in. The login page loads fine, but it hangs on login and the UI says "Connection timeout". 

Running on Unraid within docker, and using letsencrypt for my cert/nginx. Running a dozen plus other services through the proxy no problem.

**Comment by sniffingsugar** [comment]
Did you enable websocket support for your virtual host?
Please upload your nginx conf file.

Otherwise we can't help.

**Comment by mbattista** [comment]
Also there is a good documentation attached for nginx reverse proxys. Perhaps there is something in there that can help you.
https://neko.m1k1o.net/#/getting-started/reverse-proxy

**Comment by m1k1o** [comment]
Error `token not found in 'Upgrade' header` hints, that `proxy_set_header Upgrade $http_upgrade;` is missing in your proxy config.

**Comment by suckerfish** [comment]
Got it working by adding: proxy_set_header Upgrade $http_upgrade; to my existing site config.

So what's interesting is that I already tried, at one point, the exact nginx config in the documentation without success. However, I get the feeling that by the time I tried that, I had maybe already broken something else by tweaking things elsewhere, such as my port forwarding, etc. 

Thank you for your help!
